inference-engine/cmake/dependencies.cmake

```diff
@@ -46,21 +46,33 @@ endif()
 ## enable cblas_gemm from OpenBLAS package
 if (ENABLE_MKL_DNN AND GEMM STREQUAL "OPENBLAS")
     if(AARCH64)
-        reset_deps_cache(OpenBLAS_DIR)
+        if(DEFINED ENV{THIRDPARTY_SERVER_PATH})
+            set(IE_PATH_TO_DEPS "$ENV{THIRDPARTY_SERVER_PATH}")
+        elseif(DEFINED THIRDPARTY_SERVER_PATH)
+            set(IE_PATH_TO_DEPS "${THIRDPARTY_SERVER_PATH}")
+        else()
+            message(WARNING "OpenBLAS is not found!")
+        endif()
 
-        RESOLVE_DEPENDENCY(OpenBLAS
-                ARCHIVE_LIN "keembay/openblas_0.3.7_yocto_kmb.tar.xz"
-                TARGET_PATH "${TEMP}/openblas_0.3.7_yocto_kmb"
-                ENVIRONMENT "OpenBLAS_DIR")
+        if(DEFINED IE_PATH_TO_DEPS)
```

Comment:
what if `IE_PATH_TO_DEPS` is not defined? you break the old behaviour, is not it?

Reply:
`IE_PATH_TO_DEPS ` is defined below based on `THIRDPARTY_SERVER_PATH` variable. We need to define it, because this package is not available on public server. It is the same approach that is used for OpenCV on Yocto (see https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/cmake/dependencies.cmake#L182).

Reply:
@ilya-lavrenov ping